src/operator/src/insert.rs

```diff
@@ -301,7 +301,7 @@ impl Inserter {
         let request_schema = req.rows.as_ref().unwrap().schema.as_slice();
         let create_table_expr = &mut build_create_table_expr(&table_ref, request_schema)?;
 
-        info!(
+        debug!(
```

Comment:
It's better not to change from info to debug.

Reply:
ditto